src/lib/openapi/spec/admin-features-query-schema.ts

```diff
@@ -0,0 +1,26 @@
+import { FromSchema } from 'json-schema-to-ts';
+
+export const adminFeaturesQuerySchema = {
+    $id: '#/components/schemas/adminFeaturesQuerySchema',
+    type: 'object',
+    additionalProperties: true,
+    properties: {
+        tag: {
+            type: 'array',
```

Comment:
What does this parameter do? And what is it?

```suggestion
            type: 'array',
            description: 'todo',
```